docs/docs/components.mdx

```diff
@@ -1052,6 +1052,83 @@ Intent classifiers assign one of the intents defined in the domain file to incom
   ```
 
 
+### LogisticRegressionClassifier
+
+* **Short**
+
+  Logistic regression intent classifier, using the [scikit-learn implementation](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html).
```

Comment:
Not sure how pedantic we are about external links, but should this be hard-coded to the currently used version (I think it's `0.24.2`?) in case sklearn changes its implementation in future versions. On the other hand, it's quite likely that this link will not get updated if we upgrade the sklearn version either. So no real preference here 😄

Reply:
I think it's best to leave as-is. Mainly because keeping an eye on the docs is likely going to be more work and it's not unlikely that Rasa will keep sklearn updated.